src/main.rs

```diff
@@ -65,15 +61,11 @@ fn main() {
         .expect("unable to set SIGINT handler");
 
         log::debug!("connecting to server...");
-        let execution = client::execute(&args);
-        if execution.is_err() {
-            log::error!("unable to run client: {}", execution.unwrap_err());
-            std::process::exit(4);
-        }
+        client::execute(&args)?;
     } else {
         use clap::CommandFactory;
         let mut cmd = args::Args::command();
         cmd.print_help().unwrap();
-        std::process::exit(2);
```

Comment:
And this one, too. `2` is commonly used to indicate that a configuration error occurred and this exit-code is part of *rperf*'s established process interface.